### PR TITLE
Ergodox Updates

### PR DESCRIPTION
**Pull request by Xyverz**
Updating my Ergodox layout as well as bringing the syntax up to
QMK standards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: layouts/community/ergodox/xyverz/keymap.c [hunk 1]
```diff
@@ -14,36 +15,36 @@
...
 
-extern keymap_config_t keymap_config;
+enum layer_names {
+    _DVORAK,
+    _QWERTY,
+    _COLEMAK,
+    _MEDIA,
+};
+
+enum custom_keycodes { DVORAK = SAFE_RANGE, QWERTY, COLEMAK, MEDIA };
```
Comment (zvecr) [inline layouts/community/ergodox/xyverz/keymap.c]:
```suggestion
enum custom_keycodes { DVORAK = SAFE_RANGE, QWERTY, COLEMAK };
```

Comment (Xyverz) [reply layouts/community/ergodox/xyverz/keymap.c]: I've made this change. I've also added more to the comments and updated the readme.md.

**Comment by Xyverz** [comment]
It looks like three folders were added to my most recent commit. How do I remove those? They don't belong in the update and shouldn't have been included in the branch in the first place.

**Comment by Xyverz** [comment]
Something's screwy in my local repo. Closing this request. I will create another PR when I've fixed the screwiness.

**Pull request closed without merging**